engines/sci/parser/vocabulary.cpp

```diff
@@ -208,7 +208,7 @@ bool Vocabulary::loadSuffixes() {
 		suffix_t suffix;
 
 		suffix.alt_suffix = (const char *)resource->data + seeker;
-		suffix.alt_suffix_length = strlen(suffix.alt_suffix);
+		suffix.alt_suffix_length = Common::strnlen(suffix.alt_suffix, resource->size - seeker);
```

Comment:
We should maybe better error() out in case of this here when no terminating NUL is found.
Unless there is a game, that actually has such corrupted resources.